scripts/cli.sh

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/env bash
+
```

Comment:
We should add a `set -e` here so that Bash exits upon hitting an error.
I only see this being possible on a `cargo build` failure, but I can't guarantee we won't have a situation where we spawn a bunch of temporary files, then hit an error and never get a chance to clean them up.

I ran into this when my `cargo build` failed and I noticed some files were generated but never moved to the correct location. `set -e` should be fine in that case.

I think ideally, we could remove the `cargo build` entirely and just exit the script if it doesn't exist, since it's better to assume the user would build their own binary in their preferred way. However we need this to work for CI as well, so we still need to trigger a compile. Open to suggestions here.